src/templates/components/topTasks/index.test.tsx

```diff
@@ -0,0 +1,196 @@
+import { TopTasks, _topTaskLovellComp } from './index'
+import { render } from '@testing-library/react'
+
+const lovellAdministration = { entityId: 1039 }
+
+describe('TopTasks', () => {
```

Comment:
This bit is intended to make sure the component renders properly and that all the props are passed through to the `topTaskLovellComp` function, which is tested below.